### ISSUE TITLE
beforeURLChange not firing during SSR

### ISSUE THREAD

**Issue by Abourass** [roxiness/routify#362]
**Bug Description**
When I run in `dev:ssr`, `$beforeUrlChange` will fire when in SPA mode (port 5000), but doesn't fire when I run in SSR (5005). 

**Offending Code**
```javascript
$beforeUrlChange(async() => {
    await saveSettings($vendor._id, {
      preferredMapApp: $vendor.snapshotPreferences.preferredMapApp,
      pushNotifications: $vendor.snapshotPreferences.pushNotifications
    });
    return true;
  })
```

**package.json**
` "dev:ssr": "run-p routify rollup",`

**Version**
```
node: 15.12.0
npm: 7.6.3
@roxi/routify: ^2.12.4
svelte: ^3.37.0
```

**Comment by jakobrosenberg** [comment]
Can you check if the issue exists in `2.17.2`?

**Comment by Abourass** [comment]
Fixed in `2.17.2`
